encord/client.py

```diff
@@ -816,6 +859,45 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
 
         return [ProjectUser.from_dict(user) for user in users]
 
+    def get_groups(self, project_hash: str):
+        """
+        List all groups that have access to a particular project
+        """
+        return self._api_client.get(
+            f"project/{project_hash}/group", params=None, result_type=Page[OrmGroup]
+        )
+
+    def add_group(self, project_hash: str, group_param: ProjectGroupParam):
```

Comment:
I actually don't like using group_param as the parameter, I would prefer to simply separate it into it group_hash and user_role, so calling the method will look something like:
`add_group("project_hash", "group_hash", ProjectUserRole.Admin)`

What do you think?